behat_suites.yml

```diff
@@ -1,9 +1,19 @@
-cleanMeta:
+demoMeta:
```

Comment:
This one should be adapted for v2 once we're running it on Travis (the Contexts are not correct). I can prepare the correct file.

Reply:
Ok, should I do some changes here for the merge, or do we leave it wrong until you open pr afterwards?

Reply:
We can leave it, I will create another PR later.